### PR TITLE
Bug/fix promobox images

### PR DESCRIPTION
**Pull request by andygnewman**
Transfer of images transform to xslt meant that promo-box image classes were not being appended. This solves that issue for the new promo-box style (currently switched on behind flag). Will need to make happen on old promo-box style as well.

### CODE REVIEW

**Comment by matthew-andrews** [comment]
the article transform logic is getting rather complicated … i wonder if we should for the format that we get to be simplified.

thoughts?  @andygnewman @ironsidevsquincy @i-like-robots ?

File: server/transforms/images.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = function($body, opts) {
 				var $img = $body('img[data-image-set-id="' + imageSet.id.replace('http://www.ft.com/thing/', '') + '"]')
 					.attr('src', imgSrc);
 
-				if (imageSet.title) {
+				if ($img.parent().filter('figure').length > 0 && imageSet.title) {
```
Comment (i-like-robots) [inline server/transforms/images.js]: Does `.parent()` accept a selector as a filter?

Comment (andygnewman) [inline server/transforms/images.js]: @i-like-robots Unfortunately not.  .children() does, but not .parent()

**Comment by i-like-robots** [comment]
:ship:

**Pull request merged**